disnake/asset.py

```diff
@@ -34,12 +34,17 @@
 from . import utils
 from .errors import DiscordException, InvalidArgument
 
-__all__ = ("Asset",)
+__all__ = (
+    "AssetMixin",
+    "Asset",
+)
 
 if TYPE_CHECKING:
     ValidStaticFormatTypes = Literal["webp", "jpeg", "jpg", "png"]
     ValidAssetFormatTypes = Literal["webp", "jpeg", "jpg", "png", "gif"]
 
+    AssetBytes = Union[bytes, "AssetMixin"]
```

Comment:
this shouldn't be declared in TYPE_CHECKING if we're importing it into other files IMO

Reply:
We've had this problem before, with #384 iirc

Reply:
79364942bc96ce3814f3a221dd1de77c65eb001e